pdcupdater/tests/handler_tests/test_modules.py

```diff
@@ -2,11 +2,45 @@
 import shutil
 import tarfile
 import copy
+import mock
 
 from pdcupdater.tests.handler_tests import (
     BaseHandlerTest, mock_pdc
 )
 
+def mocked_koji_from_tag(url, tag):
+    if tag != 'epel7':
+        return []
+    return [{
```

Comment:
- There might be other tests for modules handler which would require mocked_koji_from_tag, but currently there is just single one. This has been actually taken from the test_rpms, where it is using the same method of testing.

- I think not all data is needed in a test (for example build_id is not used there), but it is what pdcupdater.services.koji_rpms_in_tag, so I think it makes sense to include all the data there.